pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1610,3 +1627,19 @@ func buildCompleteFilterChain(pluginParams *plugin.InputParams, mutable *plugin.
 
 	return nil
 }
+
+// getActualWildcardAndLocalHost will return corresponding Wildcard and LocalHost
+// depending on value of proxy's IPAddresses first element
+func getActualWildcardAndLocalHost(node *model.Proxy) (string, string) {
```

Comment:
It's probably ok for now - i.e. will not break anything and work in ip4-only and ip6-only.